src/flexible-text/flexible-text.component.ts

```diff
@@ -110,15 +116,30 @@ export class ThyFlexibleTextComponent implements OnInit, AfterContentInit, OnDes
 
     ngOnInit() {
         this.updateContainerClass();
-        this.tooltipService.attach(this.elementRef, this.viewContainerRef, this.trigger);
+        this.attachTooltip();
+    }
+
+    attachTooltip() {
+        this.thyTooltipDirective = new ThyTooltipDirective(
```

Comment:
this.tooltipRef = this.thyTooltipService.create(this.elementRef, {
         viewContainerRef: this.viewContainerRef,
         placement: this.placement,
         offset: this.offset,
         hasBackdrop: this.trigger === 'click',
         // ...
});